static/js/message_viewport.js

```diff
@@ -264,6 +264,11 @@ exports.system_initiated_animate_scroll = function (scroll_amount) {
     pointer.suppress_scroll_pointer_update = true; // Gets set to false in the scroll handler.
     var viewport_offset = exports.scrollTop();
     in_stoppable_autoscroll = true;
+
+    if (popovers.any_active()) {
+        return;
+    }
+
```

Comment:
I think we might want to put this in the calling function in `message_list_view.js`, and the other one in the calling function in `exports.maybe_scroll_up_selected_message`; just because that's the current structure through which we check the popover state.